peps/converters.py

```diff
@@ -181,7 +181,14 @@ def convert_pep_page(pep_number, content):
 
     data['content'] = str(pep_content)
 
-    source_link = "https://github.com/python/peps/blob/master/pep-{0}.txt".format(pep_number)
+    pep_ext = ".txt"
```

Comment:
Style nit: I'm trying to use single quotes in new code. Can we change these to single quotes?